Drop docs from setup.py, distutils-r1 takes care of it.
Handle the man pages in the ebuild because the tarball does not
contain compressed man files.

--- a/setup.py
+++ b/setup.py
@@ -13,10 +13,7 @@
         description = 'Offline Map Viewer',
         version = VERSION,
         url = WEB_ADDRESS,
-        data_files = [('share/doc/mapcatcher', ['README', 'changelog']),
-                    ('share/applications', ['gmapcatcher.desktop']),
-                    ('share/man/man1',
-                        ['man/mapcatcher.1.gz', 'man/mapdownloader.1.gz']),
+        data_files = [('share/applications', ['gmapcatcher.desktop']),
                     ('share/pixmaps', ['images/mapcatcher.png']),
                     ('share/pixmaps/gmapcatcher',
                     map(lambda(thestr): "images/" + thestr, os.listdir('images')))
